src/orders/order.controller.ts

```diff
@@ -9,12 +9,21 @@ import { ObjectId } from "mongoose";
 @Controller("orders")
 export class OrderController {
   constructor(private readonly orderService: OrderService) {}
-
+  //this route is used by cc-app
   @Get()
-  async findAll(@Query() findAllOrder: FindAllOrderDto): Promise<{ data: Order[]; total: number }> {
+  async findAll(@Query() findAllOrder: FindAllOrderDto): Promise<Order[]> {
     return await this.orderService.findAll(findAllOrder);
   }
 
+  //this route is used by app-admin
+  @Get("/admin")
+  async findAllByFilters(
+    @Query() filterDto: GetOrdersFilterDto,
+  ): Promise<{ data: Order[]; total: number }> {
+    console.log(filterDto);
+    return await this.orderService.findAllByFilters(filterDto);
+  }
```

Comment:
1. please add pagination DTO
2. please remove _console log_
3. _filterDto_ will be less clear, please use a more semantic name

Reply:
fixed